src/pods/edit-table/components/commands/commands.component.tsx

```diff
@@ -29,6 +41,30 @@ export const Commands: React.FC<Props> = (props: Props) => {
         icon={<RemoveIcon />}
         onClick={() => onDeleteField(field.id)}
       />
+      {fields[fields.length - 1].id !== field.id ? (
+        <CommandIconButton
+          icon={<DownIcon />}
+          onClick={() => onMoveDownField(field.id)}
+        />
+      ) : (
+        <CommandIconButton
+          icon={<DownIcon />}
+          onClick={() => onMoveDownField(field.id)}
+          disabled={true}
+        />
+      )}
+      {fields[0].id !== field.id ? (
+        <CommandIconButton
+          icon={<UpIcon />}
+          onClick={() => onMoveUpField(field.id)}
+        />
+      ) : (
+        <CommandIconButton
+          icon={<UpIcon />}
+          onClick={() => onMoveUpField(field.id)}
+          disabled={true}
+        />
+      )}
```

Comment:
Similar solution here:

```ts
const isFirstItemInArray = (fields : FieldVm[], fieldId :  GUID) : boolean =>
  fields[0].id !== field.id
```

And in the markup

```tsx
        <CommandIconButton
          icon={<UpIcon />}
          onClick={() => onMoveUpField(field.id)}
          disabled={isLasItemInArray(fields, field.id)}
        />
```

Reply:
It's fixed